spec/controllers/session_controller_spec.rb

```diff
@@ -83,6 +83,16 @@
     expect(subject).to redirect_to(force_user_reset_password_path(username, true))
   end
 
+  it 'Renders sign_user_form if user param is not present' do
```

Comment:
I don't think this test is addressing the sentry issue. From what I understand the Sentry error got triggered when `POST /users/sign_in` happened with no params.. so in the test we shouldn't be stubbing any cognito responses (or logging the user in) and any params....